nemo_curator/__init__.py

```diff
@@ -12,22 +12,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import dask
+try:
+    import dask
 
-from .modules import *
-from .services import (
-    AsyncLLMClient,
-    AsyncOpenAIClient,
-    LLMClient,
-    NemoDeployClient,
-    OpenAIClient,
-)
-from .utils.distributed_utils import get_client, get_network_interfaces
+    # Dask will automatically convert the list score type
+    # to a string without this option.
+    # See https://github.com/NVIDIA/NeMo-Curator/issues/33
+    # This also happens when reading and writing to files
+    dask.config.set({"dataframe.convert-string": False})
+except ImportError:
```

Comment:
Why do we need these import guards?

Reply:
We were seeing a Dask `ModuleNotFoundError` in the build and test CI, example here: https://github.com/NVIDIA/NeMo-Curator/actions/runs/11812742210/job/32908443159.

But I agree, we shouldn't have the try/except. Just debugged and figured out that it was an issue with our pyproject.toml file dynamically fetching the version from `nemo_curator/__init__.py` - switching it to fetch from the `package_info.py` file should work. Will push changes shortly.

Reply:
Thank you!